### PR TITLE
Implement `getnativecontracts` RPC

### LINKED ISSUE #1680: Implement getnativecontracts RPC call
See neo-project/neo-modules#462, it'd be nice to reuse state.Contract here, maybe refactor some common part of it out (everything but `updatecounter`) and add `activeblockindex`.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1680 .

### CODE REVIEW

**Review by AnnaShaleva — CHANGES_REQUESTED** [review]

File: pkg/core/state/contract.go [hunk 1]
```diff
@@ -30,6 +30,12 @@ type ContractBase struct {
 	Manifest manifest.Manifest `json:"manifest"`
 }
 
+// Native holds information about native contract.
+type Native struct {
```
Comment (roman-khimov) [inline pkg/core/state/contract.go:34]: `NativeContract`? `state.Native` is too broad.

File: pkg/rpc/server/server.go [hunk 2]
```diff
@@ -986,6 +987,10 @@ func (s *Server) getContractState(reqParams request.Params) (interface{}, *respo
 	return cs, nil
 }
 
+func (s *Server) getNativeContracts(_ request.Params) (interface{}, *response.Error) {
```
[the remark this answers is not in the document]
Comment (fyrchik) [reply pkg/rpc/server/server.go:990]: Fixed.

File: pkg/core/state/contract.go (hunk 1 above)
Comment (fyrchik) [reply pkg/core/state/contract.go:34]: Fixed.

**Review by roman-khimov — APPROVED** [review]
I feel a little uncomfortable without fixed-input client test, but it should be sufficient for now.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**